### PR TITLE
chore: Add directory to package.json

### PR DESCRIPTION
**Pull request by RiskyMH**
**Please describe the changes this PR makes and why it should be merged:**
This PR adds the directory property to the repository in all the `package.json`'s inside the monorepo (apps and packages).

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository
> If the `package.json` for your package is not in the root directory (for example if it is part of a monorepo), you can specify the directory in which it lives:
```json
{
  "repository": {
    "type": "git",
    "url": "https://github.com/facebook/react.git",
    "directory": "packages/react-dom"
  }
}
```

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by iCrawl — APPROVED** [review]

File: packages/scripts/src/template/template.package.json [hunk 1]
```diff
@@ -25,7 +25,9 @@
 		"lib": "src",
 		"test": "__tests__"
 	},
-	"files": ["dist"],
+	"files": [
+		"dist"
+	],
```
Comment (Jiralite) [inline packages/scripts/src/template/template.package.json:30]: Is this an unrelated change?

**Review by Jiralite — APPROVED** [review]

**Review by didinele — APPROVED** [review]
